core/controllers/reader_test.py

```diff
@@ -49,6 +49,7 @@ def test_unpublished_explorations_are_invisible_to_logged_out_users(self):
 
     def test_unpublished_explorations_are_invisible_to_unconnected_users(self):
         self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        self.login(self.VIEWER_EMAIL)
```

Comment:
Ah, I see the issue above recurs here too. The fix is optional, but if we fix the above we might as well fix this one too, I think.